jeto/static/js/controllers/domains.js

```diff
@@ -17,7 +17,7 @@ function DomainsController($scope, $routeParams, Domains, $http, createDialog, H
 
     };
     $scope.get_keys = function() {
-        $scope.domainInfo.ssl_key = null;
+        // $scope.domainInfo.ssl_key = null;
```

Comment:
Found a simple fix after all. We are already checking if there is a DC set in domainInfo. If DC is set, dont reset the domainInfo.ssl_key. If there is no DC, it means we just popup the dialog, which mean we dont want keys to be auto fetched.